docs/standard/microservices-architecture/secure-net-microservices-web-applications/index.md

```diff
@@ -294,7 +294,7 @@ The JWT bearer authentication middleware can also support more advanced scenario
   [*https://azure.microsoft.com/resources/samples/active-directory-dotnet-webapp-openidconnect-aspnetcore/*](https://azure.microsoft.com/resources/samples/active-directory-dotnet-webapp-openidconnect-aspnetcore/)
 
 - **IdentityServer4. Official documentation** \
-  [*https://identityserver4.readthedocs.io/en/release/*](https://identityserver4.readthedocs.io/en/release/)
+  *<https://identityserver4.readthedocs.io/en/latest/>*
```

Comment:
The angle brackets look good! The links above can also be converted to use this syntax.

Reply:
Hi @NextTurn, yeah, that's a relatively new standard, so I'm only applying it when updating the links!

Although it's a good candidate for a Regex global replace! 😉

Reply:
Of course! Are you going to create a PR for that? Otherwise I'll address it.

Reply:
Well, I usually arrange with @mairaw first when doing global changes, but I don't have any exclusivity for the fun tasks 😉😁 so enjoy it!

Reply:
Yep, we've been applying these to updates but it's not a high-priority task to at the moment to replace those. They'll mostly happen on the e-books content only because they follow a different convention because of the "printed" layout.